.github/workflows/integration.yml

```diff
@@ -1,6 +1,7 @@
 name: Integration testing
 env:
   PROJECT_ROOT: /root/go/src/github.com/infrawatch/sg-core
+  OPSTOOLS_REPO: https://raw.githubusercontent.com/centos-opstools/centos-release-opstools/c8s-sig-opstools/SOURCES/CentOS-OpsTools.repo
```

Comment:
```suggestion
  OPSTOOLS_REPO: https://git.centos.org/rpms/centos-release-opstools/raw/c8s-sig-opstools/f/SOURCES/CentOS-OpsTools.repo
```

Reply:
This path looks a bit more sane than pulling from github raw. Thanks for the correction!